devops/scripts/ticker

```diff
@@ -12,13 +12,24 @@ function display_dots() {
 
 function ticker() {
     local pid
+    local out
+    local status
+
+    out="$(mktemp)"
     display_dots &
     pid="$!"
-    if "$@" ; then
-        status=0
+    status=0
+    if test -n "${DOCKER_BUILD_VERBOSE:-}" ; then
+        "$@" || status=1
+    else
+        echo "Run with DOCKER_BUILD_VERBOSE=true for more information"
```

Comment:
We should also add this to the documentation, as I expect new contributors might miss this in the output but might see it in a [tip](http://docutils.sourceforge.net/docs/ref/rst/directives.html#specific-admonitions) box in the documentation, e.g. "Tip: If you're having trouble with your build, try..."